en/user/import/elCurator.md

```diff
@@ -0,0 +1,16 @@
+# From elCurator
+
+## Export your elCurator data
+
+On the settings
+([https://www.elcurator.net/settings>](https://www.elcurator.net/settings))
+page, click on "Préparer mon export" button. A JSON file will be
```

Comment:
English please!

Reply:
not sur that the interface is translated ;-)

Reply:
Homepage is when requesting English language.